cpp/src/rolling/grouped_rolling.cu

```diff
@@ -1057,29 +407,26 @@ std::unique_ptr<column> grouped_range_rolling_window(table_view const& group_key
     return optimized_unbounded_window(group_keys, input, aggr, stream, mr);
   }
 
-  using sort_groupby_helper = cudf::groupby::detail::sort::sort_groupby_helper;
-  using index_vector        = sort_groupby_helper::index_vector;
-
-  index_vector group_offsets(0, stream), group_labels(0, stream);
-  if (group_keys.num_columns() > 0) {
-    sort_groupby_helper helper{group_keys, cudf::null_policy::INCLUDE, cudf::sorted::YES, {}};
-    group_offsets = index_vector(helper.group_offsets(stream), stream);
-    group_labels  = index_vector(helper.group_labels(stream), stream);
-  }
-
-  return cudf::type_dispatcher(order_by_column.type(),
-                               dispatch_grouped_range_rolling_window{},
-                               input,
-                               order_by_column,
-                               order,
-                               group_offsets,
-                               group_labels,
-                               preceding,
-                               following,
-                               min_periods,
-                               aggr,
-                               stream,
-                               mr);
+  auto get_window_type = [](range_window_bounds const& bound) -> range_window_type {
+    if (bound.is_unbounded()) {
+      return unbounded{};
+    } else if (bound.is_current_row()) {
+      return current_row{};
+    } else {
+      return bounded_closed{bound.range_scalar()};
+    }
+  };
+  auto [preceding_column, following_column] =
+    make_range_windows(group_keys,
+                       order_by_column,
+                       order,
+                       get_window_type(preceding),
+                       get_window_type(following),
+                       stream,
+                       cudf::get_current_device_resource_ref());
+
```

Comment:
Polyfill to migrate the old interface to the new one. I propose deprecating this API in favour of the one where one passes `range_window_type`s and additionally the `null_order`.